src/murfey/client/__init__.py

```diff
@@ -16,11 +17,19 @@ def run():
     parser.add_argument(
         "--server", type=str, help="Murfey server to connect to", default=known_server
     )
-    args = parser.parse_args()
+
     parser.add_argument("--visit", help="Name of visit", required=True)
-    # visit_name = parser.parse_args().visit
+    parser.add_argument("--source", help="Directory to transfer files from")
+    parser.add_argument("--destination", help="Directory to transfer files to")
+    args = parser.parse_args()
+    visit_name = args.visit
 
-    websocket_app()
+    ws = threading.Thread(target=websocket_app)
+    ws.start()
+    if args.source and args.destination:
+        source_directory = pathlib.Path(args.source)
+        destination_directory = pathlib.Path(args.destination)
+        setup_rsync(visit_name, source_directory, destination_directory)
```

Comment:
This needs to go at the end of the function, ie. after server connectivity and update checks